spec/view_spec.cr

```diff
@@ -22,7 +22,11 @@ describe "Views" do
     end
     request = HTTP::Request.new("GET", "/view/world")
     client_response = call_request_on_app(request)
-    client_response.body.strip.should eq("<html>Hello world\n</html>")
+    {% if flag?(:windows) %}
+      client_response.body.strip.should eq("<html>Hello world\r\n</html>")
+    {% else %}
+      client_response.body.strip.should eq("<html>Hello world\n</html>")
+    {% end %}
```

Comment:
Unless the file is created on windows there should not be a difference in line endings.

Reply:
I tested this behaviour on a clone of Kemal on Windows and this change is needed for specs to pass.

Reply:
Perhaps, yet it doesn't explain the difference.

Reply:
It might help adding https://github.com/athena-framework/athena/blob/master/.gitattributes to ensure line endings remain `lf`. Ran into the same problem with athena and iirc this helped a lot since before due to my git settings it was converting everything on checkout which was incorrect.

Reply:
I really think there should not be any difference in the output format depending on whether the spec runs on Windows or Linux.
This discrepancy is most likely caused by `hello.ecr` being checked out with CRLF line ending on Windows, when it should use LF. Configuring gitattributes should fix that (probably needs rules for `*.cr` and `*.ecr`).